src/components/ButtonPanel.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Button from './Button';
+
+
+class ButtonPanel extends React.Component {
+constructor () {
+    super();
+    this.state = {
+      
+    }
+  }	
+
+  render() {
+    return (
+      <div className='buttonPanel'>
+         {
+            
+         	['AC', '+/-', '%', '+',
+             '7', '8', '9', 'X',
+             '4', '5', '6', '-',
+             '1', '2', '3', '+',
+             '0', '.', '','='].map((elm) => {
+         
+             return (<Button value = {elm}  />)
```

Comment:
As @SunnyAsar mentioned, lists in react need to have a unique `key` property in order to identify them for performance reasons. See [docs](https://reactjs.org/docs/lists-and-keys.html#keys). 
So let's add a `key` prop to the mapped component.